stl/error_formatter.py

```diff
@@ -0,0 +1,100 @@
+import abc
+import json
+import re
+
+
+class ErrorFormatter(object):
+  """Interface for displaying STL parsing errors."""
+  __metaclass__ = abc.ABCMeta
+
+  @abc.abstractmethod
+  def format(self, error):
```

Comment:
Please comment that error is type of error_handler.ErrorInfo. But, from the dependency perspective, shouldn't ErrorInfo and ErrorPosition be defined here?